staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher_whitebox_test.go

```diff
@@ -2418,13 +2418,13 @@ func TestWatchStreamSeparation(t *testing.T) {
 			}
 			// Wait before sending watch progress request to avoid https://github.com/etcd-io/etcd/issues/17507
 			// TODO(https://github.com/etcd-io/etcd/issues/17507): Remove sleep when etcd is upgraded to version with fix.
-			time.Sleep(time.Second)
+			time.Sleep(100 * time.Millisecond)
 			err = cacher.storage.RequestWatchProgress(metadata.NewOutgoingContext(context.Background(), contextMetadata))
 			if err != nil {
 				t.Fatal(err)
 			}
 			// Give time for bookmark to arrive
-			time.Sleep(time.Second)
+			time.Sleep(100 * time.Millisecond)
```

Comment:
would it make sense to `stress` the test to make sure it is not flaky ?

Reply:
Run the test couple of times, however I forgot to use `stress` let me test it again.

Reply:
(`stress` with a binary compiled with / without `-race` is most helpful)

Reply:
Done:
```
kubernetes $ rm cacher.test 
kubernetes $ go test ./staging/src/k8s.io/apiserver/pkg/storage/cacher/ --run TestWatchStreamSeparation -c -race
kubernetes $ stress ./cacher.test --test.run TestWatchStreamSeparation 
5s: 0 runs so far, 0 failures
10s: 12 runs so far, 0 failures
15s: 12 runs so far, 0 failures
20s: 24 runs so far, 0 failures
25s: 29 runs so far, 0 failures
30s: 36 runs so far, 0 failures
35s: 46 runs so far, 0 failures
```